_layouts/doc.html

```diff
@@ -5,21 +5,30 @@
       {% include header.html %}
       <main class="mdl-layout__content">
         <div class="page-content">
-          <!-- Your content goes here -->
           <div class="row">
-            <nav id="toc" style="position: fixed" class="small-12 medium-12 large-3 columns show-for-xlarge-up">
-              {% if page.dueDates %}
-                <br/>
-                <span><strong>Due Date(s):</strong> {{ page.dueDates }}</span>
+            {% capture github-link %}
```

Comment:
Since you only use ```{{github-link}}``` in one place does it need to put into it's own capture block?